tmate/install.sh

```diff
@@ -109,22 +124,40 @@ fi
 sed -i.bak "s|localhost:8080|$1|g" $SCRIPTPATH/controller.py
 sed -i.bak "s|localhost:8000|$1|g" $SCRIPTPATH/.tmate.conf
 
+# FIXME: seems like this todo is done
 # TODO: make the fun timer stuff in provision.sh work without needing to run it...
 
-# sudo install -m 755 -d /opt/hc-client
 # sudo install -m 755 $SCRIPTPATH/{connected.py,deprovision.sh,device.py,provision.sh} /opt/hc-client
 
 
-sudo install -m 755 $SCRIPTPATH/controller.py $APP_PATH
-sudo install -m 644 $SCRIPTPATH/.tmate.conf /home/$UNAME/.tmate.conf
-sudo install -m 644 $SCRIPTPATH/{session.target,session@.service,controller.service} /etc/systemd/system
+sudo -u $UNAME install -m 644 $SCRIPTPATH/.tmate.conf /home/$UNAME/.tmate.conf
+sudo install -m 755 $SCRIPTPATH/controller.py $APP_PATH/
+
+if [[ "${INIT}" == "systemd" ]]; then
+    sudo install -m 644 $SCRIPTPATH/{session.target,session@.service,controller.service} /etc/systemd/system/ || die "couldn't install systemd stuff"
+fi
+
+if [[ "${INIT}" == "upstart" ]]; then
```

Comment:
There are systems that don't use systemd :rofl:

Reply:
lol yep -- better (subjective) systems